packages/client-proxy/internal/edge/handlers/sandbox-catalog-create-entry.go

```diff
@@ -20,16 +20,24 @@ func (a *APIStore) V1SandboxCatalogCreate(c *gin.Context) {
 	body, err := parseBody[api.V1SandboxCatalogCreateJSONRequestBody](ctx, c)
 	if err != nil {
 		a.sendAPIStoreError(c, http.StatusBadRequest, fmt.Sprintf("Error when parsing request: %s", err))
-		telemetry.ReportCriticalError(ctx, "error when parsing request", err)
+		telemetry.ReportError(ctx, "error when parsing request", err)
 		return
 	}
 
 	_, span := tracer.Start(ctx, "create-sandbox-catalog-entry-handler")
 	defer span.End()
 
+	o, ok := a.orchestratorPool.GetOrchestrator(body.OrchestratorID)
+	if !ok {
+		a.sendAPIStoreError(c, http.StatusBadRequest, "Orchestrator not found")
+		telemetry.ReportError(ctx, "orchestrator not found", err)
```

Comment:
```suggestion
		telemetry.ReportError(ctx, "orchestrator not found", nil)
```

There's no error or create a new one with fmt.Errorf